test/views/CashAccountNoTransactionsV2Spec.scala

```diff
@@ -28,21 +28,50 @@ class CashAccountNoTransactionsV2Spec extends ViewTestHelper {
 
   "view" should {
 
-    "display correct contents" in new Setup {
+    "display title" in new Setup {
       val viewDoc: Document = view(model)
-
       titleShouldBeCorrect(viewDoc, "cf.cash-account.detail.title")
+    }
+
+    "display back link" in new Setup {
+      val viewDoc: Document = view(model)
```

Comment:
Could you please remove this from here and from other test cases and move to the trait Setup (and use from there) as we are using the same view (created using the same values) in all the cases?